reference-implementation/lib/abstract-ops/readable-streams.js

```diff
@@ -1287,8 +1556,8 @@ function ReadableByteStreamControllerRespondWithNewView(controller, view) {
   if (firstDescriptor.byteOffset + firstDescriptor.bytesFilled !== view.byteOffset) {
     throw new RangeError('The region specified by view does not match byobRequest');
   }
-  if (firstDescriptor.byteLength !== view.byteLength) {
-    throw new RangeError('The buffer of view has different capacity than byobRequest');
+  if (firstDescriptor.byteLength < view.byteLength) {
```

Comment:
Seems like this needs a corresponding spec change (and probably independent tests)

Reply:
I agree, this deserves its own spec change with tests. ([See also my other comment.](https://github.com/whatwg/streams/pull/1114#discussion_r600081235))

Reply:
Solved in #1123.